playground/infrastructure/cd_helper.py

```diff
@@ -82,11 +89,14 @@ def _save_to_cloud_storage(self, examples: List[Example]):
     """
     self._storage_client = storage.Client()
     self._bucket = self._storage_client.bucket(Config.BUCKET_NAME)
+    pbar = tqdm(total=len(examples))
     for example in examples:
```

Comment:
If I'm not mistaken, you can do just this:
```suggestion
    for example in tqdm(examples):
```
And then you don't need to do `pbar.update` and `pbar.close` at the end.

Reply:
Done.